crates/resolver/src/resolver.rs

```diff
@@ -232,6 +232,26 @@ impl<R: ConnectionProvider> Resolver<R> {
         .await
     }
 
+    /// Clear the cache for a specific lookup
+    ///
+    /// *WARNING* this interface may change in the future, see if one of the specializations would be better.
```

Comment:
Guess that this warning might not be relevant at this point? If there's no way to mitigate I don't think it's useful anyway.